addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14266,13 +14276,13 @@ msgstr ""
 #. Description of setting "Videos -> Library -> Flatten TV show seasons" with label #20412
 #: system/settings/settings.xml
 msgctxt "#36144"
-msgid "Remove the TV show season node, toggles between 'If only one season' (default), 'Always' and 'Never'."
+msgid "Remove the TV show season node, toggles between \"If only one season\" (default), \"Always\" and \"Never\"."
```

Comment:
I'd rephrase this particular description. In other descriptions when we explain what a particular value does, we used square braces so far. So at least in these explaintions we should be consistent. I have no strong opinion to use square braces when references as part of the text, just thought it might be more consistent. Otoh, when used in descriptions there most like is a explanation of the value. Descriptions like this particular one are plain stupid and should be rewritten anyways to be of more help.